DESCRIPTION

```diff
@@ -58,16 +58,13 @@ Suggests:
     RPostgres,
     testthat (>= 3.0.0),
     tibble,
-    simulist,
     spelling,
     usethis,
     withr
 VignetteBuilder: knitr
-Remotes:
-    github::epiverse-trace/simulist
 URL: https://github.com/ssi-dk/diseasystore, https://ssi-dk.github.io/diseasystore/
 BugReports: https://github.com/ssi-dk/diseasystore/issues
 Config/testthat/edition: 3
 Depends:
-    R (>= 2.10)
+    R (>= 3.5.0)
```

Comment:
Do we not use native piping in the codebase? This is only R >4.1.0

Reply:
It seems `tidyverse` says it prevents examples and vignettes from being run, but they don't state it as a problem in running the functions.

https://www.tidyverse.org/blog/2023/04/base-vs-magrittr-pipe/#using-the-native-pipe-in-packages